gpkit/constraints/tight.py

```diff
@@ -17,9 +17,6 @@ def __init__(self, constraints, reltol=None, raiseerror=False,
             self.reltol = reltol
         self.raiseerror = raiseerror
         self.printwarning = printwarning
-        if kwargs is not None:
-            for key, value in kwargs.items():
-                setattr(self, key, value)
```

Comment:
What do you mean @bqpd by 'parameters should be documented in interface'? I would like to be able to post-process my tight constraints by the names.

Reply:
that makes sense but I'm not sure how to connect it to this! could you give me an example? also we could probably merge this PR and bring that into another...

Reply:
Sure I can make another PR for that. The idea is to comb through the untight constraints afterwards with the purpose of adding new constraints to tighten them, or to do some other post-processing (eg. using slacks to inform some other decision).